### ISSUE TITLE
logfile timestamp should based on timezone info

### ISSUE THREAD

**Issue by TianMing2018** [paketo-buildpacks/spring-boot#298]
## Expected Behavior

timestamp echoed by console logs should based on `/etc/localtime` or `/etc/timezone`

## Current Behavior

i had changed both  `/etc/localtime` and`/etc/timezone` to `Asia/Shanghai`, but timestamp printed still used before timezone

## Steps to Reproduce

1. shells

create my own runImage and chage timezone like this
```shell
 apt-get -y install tzdata && \
  ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && \
  echo "Asia/Shanghai" /etc/timezone
```
## Motivations

timestamp should be `Asia/Shanghai` style

**Comment by scottfrederick** [comment]
> timestamp echoed by console logs

@TianMing2018 Are you talking about the logs from a Spring Boot application? If so, which logging library are you using and how are you configuring it? 

Or are you talking about the logs coming from something else?

**Comment by dmikusa** [comment]
Yes, +1. If you have an example where you could show the timestamps with some context either during build or run, that would help too.

**Comment by TianMing2018** [comment]
sorry about that, i have type wrong command which should be ` echo "Asia/Shanghai" > /etc/timezone` (miss >), which make localtime and timezone be confict and let me got wrong conjugate . I will closed this issuse and push code to <https://github.com/TianMing2018/paketo-demo>.

> By the way, there is still a problem about chinese character support . I'm not sure how to fix it, I'm terribly grateful to you for all your help.
